letta/schemas/letta_message.py

```diff
@@ -78,12 +78,14 @@ class FunctionCall(BaseModel):
 
     name: str
     arguments: str
+    function_call_id: str
```

Comment:
note to @sarahwooders @4shub we have new (required) field here for the pydantic model `LettaMessage.FunctionCallMessage`

Reply:
this field is a 1:1 with OpenAI's `tool_call_id` (for messages with role `tool`, equivalent to `FunctionReturn` in `LettaMessage`), and also `tool_call.id` (for messages with role `assistant` and `tool_calls is not None`)